Pulsarc/Utils/PulsarcTime.cs

```diff
@@ -6,7 +6,7 @@ namespace Pulsarc.Utils
     public static class PulsarcTime
     {
         // Whether or not this Class is keeping track of time.
-        public static bool IsRunning { get; private set; } = false;
+        private static bool IsRunning { get; set; }
```

Comment:
Everything you changed to private here should be changed back to public. They may not be used right now, but the intention for this class is to keep them public.